### PR TITLE
October Polaris Sync

### PR DESCRIPTION
**Pull request by Cameron653**
I didn't look what was inside. Feel free to look at what was chan-
https://i.imgur.com/RsUCKqC.png
ohboy

### CODE REVIEW

**Comment by Cameron653** [comment]
Ahhh let me fix all the problems with the PR.

**Comment by Cameron653** [comment]
THERE. IT COMPILES CORRECTLY AND EVERYTHING IS FIXED.

**Comment by Cameron653** [comment]
Tags:
Needs testing: AKA someone look over all this
Bug: Theres bugfixes in it. I know that for a fact.
Feature: Legcuffs work again.
Icon: Quite a few icons were changed
Map: Our map was changed to fix the legcuffs path. We also have randomly generated terrain (if enabled)

File: code/game/jobs/job/civilian.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 	H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/bartender(H), slot_w_uniform)
 	H.equip_to_slot_or_del(new /obj/item/device/pda/bar(H), slot_belt)
 	if(has_alt_title(H, alt_title,"Bartender"))
-		var/obj/item/weapon/permit/gun/bar/permit = new(H)
+		var/obj/item/clothing/accessory/permit/gun/bar/permit = new(H)
```
Comment (SpadesNeil) [inline code/game/jobs/job/civilian.dm]: Permits are now accessories? Make sure our permits are up to date with this change.

File: code/modules/client/preference_setup/loadout/loadout_utility.dm [hunk 2]
```diff
@@ -15,7 +15,12 @@
 
 /datum/gear/utility/codex
```
Comment (SpadesNeil) [inline code/modules/client/preference_setup/loadout/loadout_utility.dm]: this is where I left off

File: code/game/jobs/job/civilian.dm (hunk 1 above)
Comment (Cameron653) [reply code/game/jobs/job/civilian.dm]: I changed all our permits to follow as they were shooting out compile errors due to not having an owner var.

Comment (SpadesNeil) [reply code/game/jobs/job/civilian.dm]: And in the custom_items.txt as well?

File: code/modules/client/preference_setup/loadout/loadout_utility.dm [hunk 3]
```diff
@@ -15,7 +15,12 @@
...
 	display_name = "the traveler's guide to vir"
-	path = /obj/item/weapon/book/codex
+	path = /obj/item/weapon/book/codex/lore/vir
+	cost = 0
+
+/datum/gear/utility/corp_regs
+	display_name = "corporate regulations and legal code"
```
Comment (SpadesNeil) [inline code/modules/client/preference_setup/loadout/loadout_utility.dm]: I don't really want to use their corporate regs so I am likely to disable this on account of being too much of a pain in the ass to update.

File: code/modules/clothing/under/accessories/permits.dm [hunk 4]
```diff
@@ -1,40 +1,40 @@
 //This'll be used for gun permits, such as for heads of staff, antags, and bartenders
```
Comment (SpadesNeil) [inline code/modules/clothing/under/accessories/permits.dm]: File path was changed too. Make sure our file gets moved if it hadn't already.

File: code/modules/lore_codex/legal_code_data/corporate_regulations.dm [hunk 5]
```diff
@@ -0,0 +1,313 @@
+/datum/lore/codex/category/corporate_regulations
```
Comment (SpadesNeil) [inline code/modules/lore_codex/legal_code_data/corporate_regulations.dm]: This is probably going to go away.

File: code/modules/lore_codex/legal_code_data/main.dm [hunk 6]
```diff
@@ -0,0 +1,99 @@
+/datum/lore/codex/category/main_corp_regs // The top-level categories for SOP/Regs/Law/etc
```
Comment (SpadesNeil) [inline code/modules/lore_codex/legal_code_data/main.dm]: Again, this needs to go away. Sif Governmental Authority is not a thing in our lore and nor is the SCG.

File: code/modules/lore_codex/legal_code_data/sif_law.dm [hunk 7]
```diff
@@ -0,0 +1,285 @@
+/datum/lore/codex/category/sif_law
```
Comment (SpadesNeil) [inline code/modules/lore_codex/legal_code_data/sif_law.dm]: go away

File: code/modules/lore_codex/legal_code_data/sop.dm [hunk 8]
```diff
@@ -0,0 +1,348 @@
+/datum/lore/codex/category/standard_operating_procedures
```
Comment (SpadesNeil) [inline code/modules/lore_codex/legal_code_data/sop.dm]: go away, we have SOP already

File: code/modules/lore_codex/lore_data/main.dm [hunk 9]
```diff
@@ -0,0 +1,32 @@
+/datum/lore/codex/category/main_vir_lore // The top-level categories for the Vir book
```
Comment (SpadesNeil) [inline code/modules/lore_codex/lore_data/main.dm]: I don't like ANY of the changes to the lore_codex, and insist we start building our own as a replacement.

File: code/modules/mob/living/silicon/robot/robot_modules/syndicate.dm [hunk 10]
```diff
@@ -0,0 +1,45 @@
+/* Syndicate modules */
```
Comment (SpadesNeil) [inline code/modules/mob/living/silicon/robot/robot_modules/syndicate.dm]: Left off here.

File: code/game/jobs/job/civilian.dm (hunk 1 above)
Comment (Cameron653) [reply code/game/jobs/job/civilian.dm]: fuck
I need to change them in custom_items.txt. I doubt that dream daemon got them when I replaced all

File: code/modules/clothing/under/accessories/permits.dm (hunk 4 above)
Comment (Cameron653) [reply code/modules/clothing/under/accessories/permits.dm]: Ours is under /fluffstuff so I don't think it needs to be moved
https://github.com/VOREStation/VOREStation/blob/master/code/modules/vore/fluffstuff/custom_permits_vr.dm

**Comment by Cameron653** [comment]
I believe I got all the codex stuff sorted out via commenting out & unchecking the files in the .dme and leaving our edited files.

File: code/modules/client/preference_setup/loadout/loadout_utility.dm [hunk 11]
```diff
@@ -15,14 +15,14 @@
 
 /datum/gear/utility/codex
 	display_name = "the traveler's guide to vir"
-	path = /obj/item/weapon/book/codex/lore/vir
+	path = /obj/item/weapon/book/codex //VOREStation Edit
```
Comment (SpadesNeil) [inline code/modules/client/preference_setup/loadout/loadout_utility.dm]: um, this doesn't seem right

File: code/modules/lore_codex/lore_data/important_locations.dm [hunk 12]
```diff
@@ -15,7 +15,7 @@
 		/datum/lore/codex/page/rota
 		)
 
-/datum/lore/codex/page/vir/add_content()
+/datum/lore/codex/page/virgo_erigone/add_content()
 	name = "Vir (Star)"
```
Comment (SpadesNeil) [inline code/modules/lore_codex/lore_data/important_locations.dm]: This is **not** right.

File: code/modules/lore_codex/lore_data/important_locations.dm [hunk 13]
```diff
@@ -15,7 +15,7 @@
...
 	keywords += list("Vir")
 	data = "Vir is an A-type main sequence star with 81% more mass than Sol (the humans' home star), and almost nine times as bright.  It \
```
Comment (SpadesNeil) [inline code/modules/lore_codex/lore_data/important_locations.dm]: This is all wrong.

**Comment by SpadesNeil** [comment]
cameron you made it worse

File: code/modules/client/preference_setup/loadout/loadout_utility.dm (hunk 11 above)
Comment (Cameron653) [reply code/modules/client/preference_setup/loadout/loadout_utility.dm]: It is. See lore_codex/codex.dm.

/obj/item/weapon/book/codex
	var/root_type = /datum/lore/codex/category/main_vir_lore

File: code/modules/lore_codex/lore_data/important_locations.dm (hunk 12 above)
Comment (Cameron653) [reply code/modules/lore_codex/lore_data/important_locations.dm]: This is lore_data/important_locations.dm that you're looking out (which is commented out. See the .dme) not lore_data_vr/important_locations.dm. 
This file you are commenting on has no say in our current codex book but is right for the new system.

File: code/modules/lore_codex/lore_data/important_locations.dm (hunk 13 above)
Comment (Cameron653) [reply code/modules/lore_codex/lore_data/important_locations.dm]: See above

**Comment by Cameron653** [comment]
I made it better. You're commenting on commented out files ;v

**Comment by Cameron653** [comment]
Also holy fuck I started up a test server and there's a billion runtimes due to various things and everything looks to be broken on my first initial startup

File: code/modules/lore_codex/lore_data/important_locations.dm (hunk 12 above)
Comment (SpadesNeil) [reply code/modules/lore_codex/lore_data/important_locations.dm]: so wait then why is it a .dm and not _vr.dm?

Comment (SpadesNeil) [reply code/modules/lore_codex/lore_data/important_locations.dm]: -/datum/lore/codex/page/vir/add_content()
+/datum/lore/codex/page/virgo_erigone/add_content()

name = "Vir (Star)"

none of this looks right

File: code/modules/projectiles/ammunition/magazines.dm [hunk 14]
```diff
@@ -443,6 +443,23 @@
 /obj/item/ammo_magazine/m762m/empty
 	initial_ammo = 0
 
+/obj/item/ammo_magazine/m762garand
+	name = "garand clip (7.62mm)" // The clip goes into the magazine, hence the name. I'm very sure this is correct.
```
Comment (SpadesNeil) [inline code/modules/projectiles/ammunition/magazines.dm]: _yes_

**Comment by SpadesNeil** [comment]
Reviewed the rest.

File: code/modules/lore_codex/lore_data/important_locations.dm (hunk 12 above)
Comment (Cameron653) [reply code/modules/lore_codex/lore_data/important_locations.dm]: Ace. You are looking at lore_data/important_locations.dm in this comment. Not lore_data_vrr/important_locations.dm

Also, I don't know why it's not a _vr.dm. Ask @Arokha https://github.com/VOREStation/VOREStation/commits/master/code/modules/lore_codex/lore_data_vr/important_locations.dm

Comment (SpadesNeil) [reply code/modules/lore_codex/lore_data/important_locations.dm]: _Oohhhh_

Comment (SpadesNeil) [reply code/modules/lore_codex/lore_data/important_locations.dm]: nevermiiiiiind

**Pull request merged**